apps/auth_zero/tests.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from common.config import ConfigFileLoader, CredentialsLoader
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        self.user_model = get_user_model()
+        self.creds_obj = CredentialsLoader(settings.CONFIG_FILE, ConfigFileLoader)
+        self.users = {
+            verification_status: self.user_model.objects.create_user(
+                **self.creds_obj.read_config(verification_status)
+            )
+            for verification_status in [
+                "fully_unverified",
+                "only_otp_verified",
+                "only_email_verified",
+                "fully_verified",
+            ]
+        }
```

Comment:
Perhaps this should also be moved into the dev.py settings file so it does not have to be re-written for each test.

Reply:
@milan-tom  configuring for test suites for several test cases is fine since setup is done only once for all test cases;